content/guides/applications/custom-apps/jwt-setup.md

```diff
@@ -41,25 +41,32 @@ Select **Create New App**.
 Select **Custom App** from the list of application types. A modal will appear to
 prompt a selection for the next step.
 
-<ImageFrame border width="400" center>
+<ImageFrame border center>
   ![Auth selection screen](../images/select-app-type.png)
 </ImageFrame>
 
 ### 3. Select the type of authentication and app name
 
-Select **Server Authentication (with JWT)** and provide a unique name for your
-application. Click **Create App**.
+Select **Server Authentication (with JWT)** if you would like to verify 
+application identity with a key pair. Select 
+**Server Authentication (Client Credentials Grant)** if you would like to verify
+application identity with your client id and client secret. Then, provide a name
+for your application and click **Create App**.
+
+Once you make a selection, you will not be able to change to a different 
+authentication method without creating a new application.
 
 <ImageFrame border width="600" center>
   ![App name form](../images/custom-app-selection-jwt.png)
 </ImageFrame>
 
-## JWT keypair
+## Public and private key pair
```

Comment:
Since "Server Authentication (Client Credentials Grant)" doesn't have a key pair, it may be worth adding a notice message stating that this can be skipped if that method is chosen.